.vale/styles/HouseStyle/TLA.yml

```diff
@@ -0,0 +1,173 @@
+extends: conditional
```

Comment:
FYI: I found this rule to be a bit of a pain and was thinking of removing it from the blog. Lots of false positives.